lib/util/include/hse_util/vlb.h

```diff
@@ -57,7 +57,7 @@ void *vlb_alloc(size_t sz);
  */
 void vlb_free(void *mem, size_t used);
 
-merr_t vlb_init(void) HSE_COLD;
+void vlb_init(void) HSE_COLD;
```

Comment:
I think it's a cleaner model if all the init functions are plumbed to return a merr, whether or not they need to...  Makes adding code to them down the road much easier....

Reply:
I'll change it back, but you could just make this argument for function in my opinion. I don't see the difficulty in adding changing the return type 😄

Reply:
Ok for now, just an annoyance when the day comes I want to add something to the init() functions that return an error.  Used to be an extremely horrible ordeal because merr wasn't plumbed all the way down...